PROJECT_GUIDELINES.md

```diff
@@ -0,0 +1,228 @@
+# Project Guidelines
+
+Hey there and thanks for stopping by! We welcome all feedback and contributions to [Calcite App Components](https://github.com/Esri/calcite-app-components/blob/master/README.md). If you are interested in getting involved, we ask that you give our guidelines a quick read through. Below is a list of things we've found work well for us as we continue developing our components.
```

Comment:
Nitpick: `you give our guidelines a quick read through.` -> `you give our guidelines a read.`